website/source/docs/drivers/docker.html.md

```diff
@@ -246,9 +246,14 @@ This is not configurable.
 ### Authentication
 
 If you want to pull from a private repo (for example on dockerhub or quay.io),
-you will need to specify credentials in your job via the `auth` option or by
-storing the credentials in a file and setting the
-[docker.auth.config](#auth_file) value on the client.
+you will need to specify credentials in your job via:
+
+ * the `auth` option
+
+ * by storing credentials or `credHelpers` in a file and setting the
```

Comment:
Would you be willing to inline an example of what that file would look like with cred helpers set and plain base64 credentials set for a separate repo?